### DISCUSSION TITLE
Unable to pass authorization header via command prompt

### DISCUSSION THREAD

**Discussion by MandarJKulkarni** [fortio/fortio#730] [Q&A] [answered]
Here is the command I am trying with fortio 1.53.1

fortio curl -H "authorization:Bearer token" https://mywebsite.com

And the response I get is 
{"error":{"code":400,"message":"Missing authorization header","errors":[{"domain":"global","reason":"badRequest","message":"Missing authorization header"}]}}

What am I missing?

**Comment by ldemailly** [comment, marked as answer]
Not sure, something is wrong with your token or your backend, see:

```
% fortio curl -H "authorization:Bearer token" http://debug.fortio.org
10:35:48 I scli.go:88> Starting Φορτίο 1.53.1 h1:aeAWrnkXIBEdhueQLRwvaM5INulybDP+ZgOChgKz/NQ= go1.20.2 arm64 darwin
HTTP/1.1 200 OK
Content-Type: text/plain; charset=UTF-8
Date: Sat, 11 Mar 2023 18:35:48 GMT
Content-Length: 319

Φορτίο version 1.53.0 h1:OF6ArR3f7UEYqPsr357WTk8AFWKQ38Xz7wiBipK8ou4= go1.19.6 amd64 linux (in fortio.org/proxy 1.12.0)
Debug server on ol1 up for 136h42m29s
Request from 99.73.39.83:63507

GET / HTTP/1.1

headers:

Host: debug.fortio.org
Authorization: Bearer token
User-Agent: fortio.org/fortio-1.53.1

body:

```

**Reply by ldemailly** [reply]
There is also:
```
% fortio curl -user foo:bar http://debug.fortio.org
10:37:11 I scli.go:88> Starting Φορτίο 1.53.1 h1:aeAWrnkXIBEdhueQLRwvaM5INulybDP+ZgOChgKz/NQ= go1.20.2 arm64 darwin
HTTP/1.1 200 OK
Content-Type: text/plain; charset=UTF-8
Date: Sat, 11 Mar 2023 18:37:11 GMT
Content-Length: 327

Φορτίο version 1.53.0 h1:OF6ArR3f7UEYqPsr357WTk8AFWKQ38Xz7wiBipK8ou4= go1.19.6 amd64 linux (in fortio.org/proxy 1.12.0)
Debug server on ol1 up for 136h43m52.2s
Request from 99.73.39.83:63550

GET / HTTP/1.1

headers:

Host: debug.fortio.org
Authorization: Basic Zm9vOmJhcg==
User-Agent: fortio.org/fortio-1.53.1

body:

```

**Reply by MandarJKulkarni** [reply]
I am trying this with windows executable... I don't think it matters, does it?

**Reply by ldemailly** [reply]
it shouldn't matter except maybe for how that cli/shell quotes strings - try the above (with a test/invalid token) against https://debug.fortio.org and see/paste the result?

**Reply by MandarJKulkarni** [reply]
![image](https://user-images.githubusercontent.com/33712629/224531987-f2e1475e-c639-44d2-8653-edb8fd3c2013.png)
It worked for debug.fortio.org.

Let me analyze my server. Thank you for quick responses.
